docs/SupportedONNXOps-NNPA.md

```diff
@@ -12,32 +12,32 @@ Onnx-mlir currently supports ONNX operations targeting up to opset 18. Limitatio
-| **GlobalAveragePool** |1 |- Input shape must be 4D tensor(NCHW).<br>- Dimensions in `H` and `W` must be static. | |
-| **LSTM** |14 |- `direction` and `hidden_size` in `W` must have static dimensions.<br>- `R` must have static dimensions.<br>- `B` and `initial_h` have static dimensions if given. `B`'s direction dim must be 1 or 2.<br>- `P`(peepholes), `activation_alpha`, and `activation_beta` are not supported.<br>- `activations` must be `["Sigmoid", "Tanh", "Tanh"]`.<br>- `clip` is not supported.<br>- `input_forget` must be default value(0).<br>- `layout` is not supported. | |
-| **LeakyRelu** |16 |The operations immediately before and after the LeakyRelu operation must be executed on the NNPA. Otherwise, LeakyRelu is executed on the CPU. This limitation is set to avoid performance degradation. | |
-| **Log** |13 |Input tensor must have 4 dimensions. | |
-| **LogSoftmax** |13 | | |
-| **MatMul** |13 |Ranks of input tensors must be (Rank of A, Rank of B) = (M, N), where M >= 2 and N >= 2. | |
-| **Max** |13 |- Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **MaxPool** |12 |- `auto_pad` must be `NOTSET`, `VALID`, and `SAME_UPPER`. If `NOTSET` is used, `pads` must be set so that the padding valid type or same upper.<br>- `ceil_mode` must be default value(0) <br>- Input and output tensors must be 4D tensors(N x C x H x W).<br>- `kernel_shape` must be static.<br>- `ceil_mode` must be default value(0).<br>- `dilations` must be default value(1). | |
-| **Min** |13 |- Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **Mul** |14 |- Shape of input tensors should be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **Pow** |15 |- Exponent should be a scalar integer and less or equal to 64. | |
-| **ReduceMean** |18, 13 |- `keepdims` must be 1.<br>- Input tensor must be 4D tensors and `axis` must be [2, 3]. | |
-| **Relu** |14 |Input tensor must be less than or equal to 4 dimensions. | |
-| **Sigmoid** |13 |Input tensor must be less than or equal to 4 dimensions. | |
-| **Softmax** |13, 11 |- `axis` must be the last dimension, i.e. `rank - 1` or -1. | |
-| **Sub** |14 |- Shape of input tensors should be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
-| **Sum** |13 |- All inputs must have the same static shape (Broadcasting not supported.)<br>- Single input not supported. | |
-| **Tanh** |13 |Input tensor must be less than or equal to 4 dimensions. | |
+| Op |Min Opset |Max Opset |Limitations |Notes |
+| --- |--- |--- |--- |--- |
+| **Add** |14 |18 |- Shape of input tensors must be the same since broadcasting is not supported.<br>- Input tensors must have static dimensions. | |
```

Comment:
@hamptonm1 Why 18 for this? Shouldn't it either be 19 (the latest OpSet this onnx-mlir supports), 14 (the last updated version of Add) or * (if we go with my other comment).

Reply:
Since I did not specify a `MAX` it defaulted to 18. @AlexandreEichenberger set the default min to 9 and the default max to 18. Given we predominately have support for those versions. From our conversations in the past.... I think we can change it to make the min 8 and the max 19. However, I will leave this to Alex as well :)

Reply:
As soon as we migrate to a new level, say 19, we should put the default to 19, and every op that we don't support yet should have their max then set to 18. As we then add support for 19, removing that max (removing it entirely) will be part of the PR adding the new support.

Reply:
@cjvolzka I think a conversation offline should be had based on the comments. I understand what you mean but @AlexandreEichenberger explained to me his reasoning and how others in the community interpret the opset list. I will schedule some time to talk but I definitely understand Alex's logic after talking with him.